src/explore-education-statistics-common/src/modules/release/components/BasicReleaseSummary.tsx

```diff
@@ -0,0 +1,127 @@
+import { Dictionary } from '@common/types';
+import Modal from '@common/components/Modal';
+import { parseISO } from 'date-fns';
+import React, { ReactNode } from 'react';
+
+interface ReleaseTypeIcon {
+  url: string;
+  altText: string;
+}
+
+const nationalStatisticsLogo: ReleaseTypeIcon = {
+  url: '/assets/images/UKSA-quality-mark.jpg',
+  altText: 'UK statistics authority quality mark',
+};
+
+const releaseTypesToIcons: Dictionary<ReleaseTypeIcon> = {
+  NationalStatistics: nationalStatisticsLogo,
+};
+
+interface Props {
+  isEditing?: boolean;
+  lastUpdated?: Date;
+  release: Release;
+  releaseDate: string;
+  renderReleaseNotes: ReactNode;
+  renderStatusTags: ReactNode;
+  renderSubscribeLink: ReactNode;
+  onToggleReleaseTypeModal?: () => void;
+}
+
+export default function BasicReleaseSummary({
+  isEditing,
+  lastUpdated,
+  release,
+  releaseDate,
+  renderReleaseNotes,
+  renderStatusTags,
+  renderSubscribeLink,
+  onToggleReleaseTypeModal,
+}: Props) {
+  const [showReleaseTypeModal, toggleReleaseTypeModal] = useToggle(false);
+
+  const ReleaseTypeComponent = releaseTypeComponents[release.type];
+
+  return (
+    <>
+      <div className="dfe-flex dfe-align-items--center dfe-justify-content--space-between govuk-!-margin-bottom-3">
+        <div>{renderStatusTags}</div>
+        {releaseTypesToIcons[release.type] && (
+          <img
+            src={releaseTypesToIcons[release.type].url}
+            alt={releaseTypesToIcons[release.type].altText}
+            height="60"
+            width="60"
+          />
+        )}
+      </div>
+
+      <SummaryList>
+        <SummaryListItem term="Published">
+          {releaseDate ? (
+            <FormattedDate>{parseISO(releaseDate)}</FormattedDate>
+          ) : (
+            <p>TBA</p>
+          )}
+        </SummaryListItem>
+        {release.latestRelease &&
+          isValidPartialDate(release.nextReleaseDate) && (
+            <SummaryListItem term="Next update">
+              <time>{formatPartialDate(release.nextReleaseDate)}</time>
+            </SummaryListItem>
+          )}
+
+        {(isEditing || lastUpdated) && (
+          <SummaryListItem term="Last updated">
+            {lastUpdated && <FormattedDate>{lastUpdated}</FormattedDate>}
+            {renderReleaseNotes}
+          </SummaryListItem>
+        )}
+
+        <SummaryListItem term="Release type">
+          <ButtonText
+            onClick={() => {
+              toggleReleaseTypeModal.on();
+              onToggleReleaseTypeModal?.();
+            }}
+          >
+            {releaseTypes[release.type]}{' '}
+            <InfoIcon
+              description={`Information on ${releaseTypes[release.type]}`}
+            />
+          </ButtonText>
+        </SummaryListItem>
+
+        <SummaryListItem term="Receive updates">
+          {renderSubscribeLink}
+        </SummaryListItem>
+      </SummaryList>
+
+      <Modal
+        className="govuk-!-width-one-half"
```

Comment:
Not sure this responds too great on smaller desktops / tablets: 

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/9917868/79183cd4-7d83-4ce4-b073-ba4814693362)

It doesn't seem like we really need this at all personally.

Without the width:

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/9917868/29f4a781-6c7c-4f20-8d75-3ca3e3787e71)

With the width:

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/9917868/a55c5d7e-3f0c-48ea-a3bf-63f31f89827e)